### PR TITLE
Reboot command will not be applicable to certification of DUTs adding user prompt to manually reboot the DUT

### PR DESCRIPTION
**Pull request by manjunath-grl**
The reboot mechanism we have in YAML might work for our SDK CI , since its based on opening a socket on port 9000 and doing a xpc connection but don't think it will be applicable to certification of DUTs, removing reboot command all steps where we use reboot to a user prompt to manually reboot
TC-BINFO-2.2
TC-DGEN-2.1
TC-WNCV-4.5
Modified scripts 
TC-MF-1.3
TC-MF-1.4
TC-MF-1.5
TC-MF-1.6
TC-MF-1.9
TC-MF-1.10
TC-MF-1.15
added auto generated files

Attached execution logs for reference:
[Execution_log.txt](https://github.com/project-chip/connectedhomeip/files/9019083/Execution_log.txt)

### CODE REVIEW

File: src/app/tests/suites/certification/Test_TC_MF_1_4.yaml [hunk 1]
```diff
@@ -28,20 +28,6 @@ config:
...
-          "Start target device with the provided discriminator for basic
-          commissioning advertisement"
-      cluster: "SystemCommands"
-      command: "Start"
-      arguments:
-          values:
-              - name: "discriminator"
-                value: discriminator
-
     - label: "TH_CR1 starts a commissioning process with DUT_CE"
```
Comment (raju-apple) [inline src/app/tests/suites/certification/Test_TC_MF_1_4.yaml:31]: Why not adding a prompt in the MF test cases ?

**Review by bzbarsky-apple — CHANGES_REQUESTED** [review]
The right fix is to leave Reboot for CI, but have a prompt when not in CI.

**Review by woody-apple — APPROVED** [review]

File: src/app/tests/suites/certification/Test_TC_MF_1_4.yaml (hunk 1 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_MF_1_4.yaml:31]: @raju-apple as per the MF testplan we don't need need reboot command

**Comment by raju-apple** [comment]
> The right fix is to leave Reboot for CI, but have a prompt when not in CI.

Makes sense actually , why remove some of the functionality we already have :) @manjunath-grl lets do this , for every step for reboot that we just disabled lets do this instead : 
1. Have both methods to reboot , for CI and the manual user prompt. Lets just have 2 PICS in there to differentiate.
2. So for the steps with reboot using manual user prompt , use the PICS_SKIP_SAMPLE_APP condition to make sure it gets skipped in CI but gets picked up by the TH.
3. For the existing steps which we just disabled in this PR , create a new PICS_SDK_CI_ONLY and add it to the ci-pics-value as well and make it default to 1. That way it will run in CI , I will file and issue once you PR is up on the TH to make sure to skip the automated reboot steps when TH runs it.

Let me know if this makes sense and update the PR.

**Review by bauerschwan — APPROVED** [review]

**Review by bzbarsky-apple — APPROVED** [review]
Approved assuming the TC_MF scripts that are having the stop/start removed have been run using the CI harness script and pass.

**Comment by woody-apple** [comment]
Fast tracking test updates.

**Pull request merged**